src/Components/Room.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
+import UserTile from "./UserTile";
 
 
 const Room = ({ room = {}, handleLeaveRoom, handleStartGame }) =>{
   const { users, dayCount } = room;
   const gameReady = users && users.length > 2;
+  const userTiles = (users || []).map((u, index) => <UserTile user={u} key={index}></UserTile>);
```

Comment:
Looks great!
You don't need a closing tab with react components (unless you're enclosing another component within them)

Reply:
Ah cool good to know!